### ISSUE TITLE
Profiles empty

### ISSUE THREAD

**Issue by ianrussel** [dbrent-amazon/amazon-advertising-api-python#4]
Hi @dbrent-amazon , I came to this page because I need some help. I already read the amazon advertising docs but there's no part of the docs that help me solve my issue. I followed everything, from setting up the account access to all apis, and even implemented the LWA in frontend side. Using sandbox its all good, since sandbox allows to create a profile(both seller and vendor) I can test all endpoint to get dummy data using the created profileid in sandbox. But when I switch to real endpoint, the profiles has no result. I have a potential customer with seller account and we tried to test the LWA in our frontend, it authorised my app, no errors returned, and it return the correct profile information about the seller, but I dont see any information that relates to a profile id. The only fields it returned are customer id, email and name(its also in LWA docs, so the returned data are correct). I can confirm my potencial customer has campaigns, hes doing it for more than a year, and he even give me  access to his google data studio dashboard that shows all his data, like Sponsored Products, Sponsored Brand, search terms etc. Is there any additional steps that needs to be done? Thanks

**Comment by saleweaver** [comment]
Did you find a solution to this @ianrussel ?

**Comment by ianrussel** [comment]
@saleweaver yes. If you encountered same issue you need to create a ticket in thier jira support because it is related to setting your credentials
